securesystemslib/rsa_keys.py

```diff
@@ -278,6 +327,10 @@ def create_rsa_signature(private_key, data, scheme='rsassa-pss-sha256'):
     scheme:
       The signature scheme used to generate the signature.
 
+    salt_length_type:
+      The strategy for determining the length of the salt used in RSA-PSS, one of
```

Comment:
It's a bit ironic that there are plenty of unrelated re-wraps in your diff to match the prescribed line length, but a line that you added exceeds it.

I wish it was the other way around. Could you please remove non-related style changes from this PR as requested in [an earlier review](https://github.com/secure-systems-lab/securesystemslib/pull/262#pullrequestreview-471518591).